ext/hal/nordic/CMakeLists.txt

```diff
@@ -50,4 +50,5 @@ if(CONFIG_HAS_NRFX)
   zephyr_sources_ifdef(CONFIG_NRFX_UARTE   nrfx/drivers/src/nrfx_uarte.c)
   zephyr_sources_ifdef(CONFIG_NRFX_USBD    nrfx/drivers/src/nrfx_usbd.c)
   zephyr_sources_ifdef(CONFIG_NRFX_WDT     nrfx/drivers/src/nrfx_wdt.c)
+  zephyr_sources_ifdef(CONFIG_NRFX_NVMC    nrfx/drivers/src/nrfx_nvmc.c)
```

Comment:
Keep the alphabetical order, please. Same applies to [Kconfig](https://github.com/NordicPlayground/fw-nrfconnect-zephyr/pull/118/files#diff-9669e335226950e2e969596610894b4dR129).